src/translations.js

```diff
@@ -220,6 +226,7 @@ const repoCardLocales = {
     np: "टेम्पलेट",
     el: "Πρότυπο",
     ru: "Шаблон",
+    ua: "Шаблон",
```

Comment:
I think `ua` is incorrect since it specifies Ukraine, not the Ukrainian language.

Reply:
But for example, ru or el, doesn't it indicate the country or something?
I'm not going to argue with you, that's all.